ci/docker/runtime_functions.sh

```diff
@@ -83,31 +102,40 @@ build_armv6() {
         -DBUILD_CPP_EXAMPLES=OFF \
         -Dmxnet_LINKER_LIBS=-lgfortran \
         -G Ninja /work/mxnet
+
     ninja
-    export MXNET_LIBRARY_PATH=`pwd`/libmxnet.so
-    cd /work/mxnet/python
-    python setup.py bdist_wheel --universal
-    cp dist/*.whl /work/build
+    build_wheel
+
     popd
 }
 
 build_armv7() {
     set -ex
     pushd .
     cd /work/build
-    cmake\
-        -DUSE_CUDA=OFF\
-        -DUSE_OPENCV=OFF\
-        -DUSE_OPENMP=OFF\
-        -DUSE_SIGNAL_HANDLER=ON\
-        -DCMAKE_BUILD_TYPE=RelWithDebInfo\
-        -DUSE_MKL_IF_AVAILABLE=OFF\
+
+    # Lapack functionality will be included and statically linked to openblas.
```

Comment:
This sounds a bit like a hack to me. Could we have proper handling for that? Otherwise, there might be other things that depend on the USE_LAPACK flag that are not getting triggered because it's expected to be absent while it's actually there.

Reply:
Sure, I already tried that and this will not be easy. I suggest this will be part of a separate PR.

Reply:
I'd prefer to have this within this PR as otherwise would leave the two branches in a semi-consistent state in terms of the USE_LAPACK flag until you follow up with the second PR.

Reply:
As I've said already, it is not a trivial task and can take up significant time and effort with functionality change impacting multiple builds. This PR is about fixing the ARMv7 build, for now without improvements. It is a big comment and only a few lines to keep in sync, so that it can be considered as tolerated evil. There is already a ticket to track the work on the problem: [MXNET-115](https://issues.apache.org/jira/browse/MXNET-115)

Reply:
Alright. Since I'm not very familiar with the impact of the LAPACK-Flag (or it being absent), maybe @szha @eric-haibin-lin @piiswrong could help out here?

Reply:
How does arm7 library get its lapack?

Reply:
how do you get libgfortran? We use -fPIC to compile libmxnet, while the libgfortran on apt wasn't compiled with -fPIC. this probably means you need to compile gfortran from source.

Reply:
Marco I don't get your point.

Reply:
@szha I don't get you point assuming you meant [position-independent code (PIC)](https://gcc.gnu.org/onlinedocs/gcc/Code-Gen-Options.html#Code-Gen-Options) flag. It is important to understand that libgfortran comes from the cross compilation lib folder `/usr/lib/arm-linux-gnueabihf/libgfortran.so.3.0.0` and it's not a host library.

Reply:
[Here](https://github.com/xianyi/OpenBLAS/issues/296) is a proof that OpenBLAS does not have a separate lapack library.

Reply:
It’s not about openblas itself, but rather whether the library requires gfortran at runtime or not. If not, then a libgfortran’s .a file that was compiled with -fPIC is required. You mentioned static linking in the comment so I assumed that you meant a complete static linking.

Reply:
Static linking goes with openblas, but fortran needs to be linked dynamically. [Comment](https://github.com/xianyi/OpenBLAS/issues/460#issuecomment-61293128) about `-static-libgfortran` not working properly

Reply:
Anyway we will look into the mentioned issue above, no doubt. Do you guys see it as possible to introduce the improvement in a follow up PR?

Reply:
@larroy I was talking about this code: https://github.com/apache/incubator-mxnet/blob/c4a76da62fbc5e3e7272fd89294fba6b22868bba/src/operator/c_lapack_api.h#L218

It seems like @lebeg and @larroy are both working on the USE_LAPACK flag. While this PR assumes the "wrong" behaviour of our build system to basically ignore the flag (as described in the ticket), https://github.com/apache/incubator-mxnet/pull/11094 attempts to correct this. I think we should first get https://github.com/apache/incubator-mxnet/pull/11094 merged properly. After that, we actually have a USE_LAPACK behaviour that behaves like expected. This would allows us to remove the hacks we have in this PR (USE_LAPACK=0) and thus reach a clean state.

If we merge this PR first and https://github.com/apache/incubator-mxnet/pull/11094 gets merged afterwards, the LAPACK build is broken again. I'd prefer a clean solution over multiple PRs about the same issue.

Reply:
As discussed, I'm working on a fix that will be introduced as a separate PR

Reply:
Ok